### PR TITLE
Optimization blog Part2

### PR DESCRIPTION
**Pull request by carl-mastrangelo**
@hsaliak and @LisaFC  FYI

cc @ncteisen

### CODE REVIEW

File: _posts/2018-04-16-so-you-want-to-optimize-grpc-part-2.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+How fast is gRPC?  Pretty fast if you understand how modern clients and servers are built.  In
+[part 1](/blog/optimizing-grpc-part-1), I showed how to get an easy **60%** improvement.  In this 
+post I show how to get a **10000%** improvement.
+
+<!--more-->
+
+## Setup
+
+As in [part 1](/blog/optimizing-grpc-part-1), we will be starting with an existing, Java based,
```
Comment (LisaFC) [inline _posts/2018-04-16-so-you-want-to-optimize-grpc-part-2.md]: we will be starting -> we will start

File: _posts/2018-04-16-so-you-want-to-optimize-grpc-part-2.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+How fast is gRPC?  Pretty fast if you understand how modern clients and servers are built.  In
+[part 1](/blog/optimizing-grpc-part-1), I showed how to get an easy **60%** improvement.  In this 
+post I show how to get a **10000%** improvement.
+
+<!--more-->
+
+## Setup
+
+As in [part 1](/blog/optimizing-grpc-part-1), we will be starting with an existing, Java based, 
+key-value service.  The service will be offering concurrent access for creating, reading, updating,
```
Comment (LisaFC) [inline _posts/2018-04-16-so-you-want-to-optimize-grpc-part-2.md]: The service will be offering -> the service will offer

File: _posts/2018-04-16-so-you-want-to-optimize-grpc-part-2.md (hunk 1 above)
Comment (carl-mastrangelo) [reply _posts/2018-04-16-so-you-want-to-optimize-grpc-part-2.md]: Done

File: _posts/2018-04-16-so-you-want-to-optimize-grpc-part-2.md (hunk 2 above)
Comment (carl-mastrangelo) [reply _posts/2018-04-16-so-you-want-to-optimize-grpc-part-2.md]: Done.

**Comment by carl-mastrangelo** [comment]
@LisaFC @hsaliak PTAL

**Comment by carl-mastrangelo** [comment]
@LisaFC @hsaliak Another friendly ping

**Pull request merged**